digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/ManoeuvreService.scala

```diff
@@ -152,13 +152,29 @@ class ManoeuvreService(roadLinkService: RoadLinkService, eventBus: DigiroadEvent
       ManoeuvreElement(0, linkPair._1, linkPair._2, ElementTypes.IntermediateElement)
     )
 
+    if (isValidLinkChain(linkPairs) == false) return false
```

Comment:
eikös toi == false oo turha
```suggestion
    if (!isValidLinkChain(linkPairs)) return false
```

Reply:
tyylillisesti vähän outoo heitää return tässä mutta kyl toimii tässä tilanteessa, scala return on sellainen vältettävä sana, https://blog.knoldus.com/scala-best-practices-say-no-to-return/

Reply:
toi sun saattaa olla vähän parempi, tää taitaa olla tälläinen miten haluu tehdä homman